.github/workflows/heater-shaker-create-release.yaml

```diff
@@ -0,0 +1,109 @@
+name: 'Heater-shaker create release from tag'
+on:
+  push:
+    tags:
+      - 'heater-shaker@*'
+  workflow_dispatch:
+
+
+defaults:
+  run:
+    shell: bash
+
+
+jobs:
+  cross-compile-check-and-release:
+    name: 'Cross-Compile/Check and Create Release'
+    runs-on: 'ubuntu-20.04'
+    timeout-minutes: 10
+    steps:
+      - uses: 'actions/checkout@v2'
+        with:
+          fetch-depth: 0
+      - uses: 'actions/cache@v2'
+        with:
+          key: stm32-cross-${{ secrets.MODULES_STM32_CACHE_VERSION }}-${{ hashFiles('cmake/*') }}
+          restore-keys: stm32-cross-${{ secrets.MODULES_STM32_CACHE_VERSION }}-
+          path: './stm32-tools'
+      - name: 'Configure'
+        run: cmake --preset=stm32-cross .
+      - name: 'Format'
+        run: cmake --build ./build-stm32-cross --target heater-shaker-format-ci
+      - name: 'Lint'
+        run: cmake --build ./build-stm32-cross --target heater-shaker-lint
+      - name: 'Build'
+        run: cmake --build ./build-stm32-cross --target heater-shaker-hex
+      - name: 'Build Combo Image Hex'
+        run: cmake --build ./build-stm32-cross --target heater-shaker-image-hex
+      - name: 'Build Combo Image Binary'
+        run: cmake --build ./build-stm32-cross --target heater-shaker-image-bin
+      - name: 'Create Release'
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          body: |
+            Changes in this Release
+            - change list
```

Comment:
probably should either actually generate a change list, possibly even by subshelling out to `git log ./stm32-modules/heater-shaker`, or get rid of this